apps/web/package.json

```diff
@@ -23,7 +23,7 @@
     "@hookform/resolvers": "^3.9.0",
     "@next-auth/prisma-adapter": "^1.0.5",
     "@planetscale/database": "^1.18.0",
-    "@prisma/adapter-planetscale": "5.18.0",
+    "@prisma/adapter-planetscale": "5.14.0",
```

Comment:
Had to downgrade `@prisma/adapter-planetscale` to get unicode characters to work properly in the `lander` JSON - seems to be related to [this issue](https://github.com/prisma/prisma/issues/24592).

Not sure if this will cause us any other problems...

Reply:
sent feedback in slack! feel free to resolve this once you change to `prism`